source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,322 @@
+Bulk export data
+=================
+At this time, core properties of non-deleted objects listed below are exported.  Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. 
```

Comment:
Wondering how obvious the first sentence is for non-technical admins, or for admins with a non-English first language

Reply:
for the second sentence, would we want to direct feature proposals to our feature idea forum https://mattermost.uservoice.com/forums/306457-general ?

Reply:
revisions made